ios/Podfile.lock

```diff
@@ -911,4 +911,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: f8a1b9d32ad4447b452262ee2a33f0ffd98f6d4d
 
-COCOAPODS: 1.12.0
```

Comment:
I think this file shouldn't have changed

Reply:
It is okay to increase cocoapods version